railties/lib/rails/generators/rails/app/templates/config/databases/mysql.yml.tt

```diff
@@ -6,11 +6,14 @@
 # Ensure the MySQL gem is defined in your Gemfile
 #   gem "mysql2"
 #
+# Ensure the Active Record is configured to use mysql2
+#   config.active_record.mysql_adapter = :mysql2
+#
```

Comment:
I do't think we need that. My vision was that you are still free to use `trilogy` or `mysql2` everywhere, we just add a dynamic `mysql` alias, that defaults to `trilogy`.